### ISSUE TITLE
[New feature]Add local dns records to values file

### ISSUE THREAD

**Issue by i5Js** [MoJo2600/pihole-kubernetes#131]
Starting in 5.0, we can add local records to Pihole using the web interface. Theses entries are save in the file /etc/pihole/custom.list.

Is it possible to add it?

Thanks.

**Comment by MoJo2600** [comment]
Yes, this should be possible. We have to make sure, that the file is stored in the persisted volume.

**Comment by i5Js** [comment]
Ok, thanks!

**Comment by MoJo2600** [comment]
@i5Js is this working for you? I'd like to close the issue if so.

**Comment by i5Js** [comment]
Hi, 
I've checked values.yml, but I can't find where I should add the values for custom.list

**Comment by ngrigoriev** [comment]
There is dnsmasq.customDnsEntries variable, it allows to inject address=/host/IP entries to dnsmasq config. Although these won't show in GUI.
